rllib/BUILD

```diff
@@ -1598,6 +1598,13 @@ py_test(
     srcs = ["offline/estimators/tests/test_dr_learning.py"],
 )
 
+py_test(
```

Comment:
awesome!

super nit: let's keep test cases alphabetically ordered (within their category), so let's move this below the next record (`test_offline_data`).